# message type for vehicle position

float64 position
float64 velocity

